### PR TITLE
Full review done

### PR DESCRIPTION
**Pull request by vmmiller**
1) Replaced "personnes handicapés" with "personnes en situation de handicap"
2) Replaced "L'objectif premier" with "L'objectif principal"
3) Replaced "après l’implémentation de chaque étape" with "après l'implémentation de chaque fonctionnalité" (E version "After there are implementations of each feature" (rather than "step" (F))
4) Replaced "Il s'agit de" with "C'était .." (E version "That was ..)
5) Replaced "de personnes handicapées" with "personnes en situation de handicap"

### CODE REVIEW

**Comment by vmmiller** [comment]
@shawna-slh - okay to merge. Completed review of the full resource.

**Comment by mtrackeros** [comment]
> 1. Replaced "personnes handicapés" with "personnes en situation de handicap"
> 2. Replaced "L'objectif premier" with "L'objectif principal"
> 3. Replaced "après l’implémentation de chaque étape" with "après l'implémentation de chaque fonctionnalité" (E version "After there are implementations of each feature" (rather than "step" (F))
> 4. Replaced "Il s'agit de" with "C'était .." (E version "That was ..)
> 5. Replaced "de personnes handicapées" with "personnes en situation de handicap"

**Review by mtrackeros — APPROVED** [review]

**Review by mtrackeros — COMMENTED** [review]
content/index.fr.md

File: content/index.fr.md [hunk 1]
```diff
@@ -78,12 +78,12 @@ Les étapes importantes par lesquelles doit passer le "rapport technique" pour d
...
 5.  {:#rec}![approved technical report:]({{ "/content-images/wai-about-w3c-process/rec.gif" | relative_url }}){:style="float:left; margin-right: 1em; clear: left;"} **Recommandation du W3C
     (Standard Web)** : Lorsque le rapport technique est approuvé par les membres, le directeur et le public du W3C, il est publié en tant que recommandation. Le W3C encourage le déploiement à grande échelle de ses recommandations.
 
-Il s'agit d'une description simplifiée du processus. Pour la version définitive, consultez le [document sur le processus du W3C, section 7 : Processus de développement d'un rapport technique du W3C](http://www.w3.org/Consortium/Process/#Reports).
+C'était une description simplifiée du processus. Pour la version définitive, consultez le [document sur le processus du W3C, section 7 : Processus de développement d'un rapport technique du W3C](http://www.w3.org/Consortium/Process/#Reports).
```
Comment (remibetin) [inline content/index.fr.md:86]: @vmmiller I tend to prefer "Il s'agit d'une description simplifiée du processus", that conveys the message "You have just read a simplified description of the process". That is what I understand when I read "That was a simplified description of the process".

In that context, "C'était une description simplifiée du process" sounds a bit too informal. Don't you think ?

**Pull request merged**